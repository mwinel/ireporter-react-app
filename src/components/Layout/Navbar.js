import React, { Component } from 'react';

class Navbar extends Component {
  render () {
    return (
      <div>
        <div className="nav">
          <div className="container">
            <div className="brand">
              <a href="./">iReporter</a>
            </div>
            <label htmlFor="toggle">&#9776;</label>
            <input type="checkbox" id="toggle" />
            <div className="menu">
              <a href="./signup">signup</a>
              <a href="#">login</a>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

export default Navbar;
