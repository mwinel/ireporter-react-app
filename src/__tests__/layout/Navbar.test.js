import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Navbar from '../../components/Layout/Navbar';

describe('Navbar Component', () => {
  let NavbarComponent = shallow(<Navbar />)
  it('Should match snapshot', () => {
    expect(toJson(NavbarComponent)).toMatchSnapshot();
  });
});
