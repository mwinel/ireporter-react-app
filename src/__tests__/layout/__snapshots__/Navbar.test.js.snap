// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component Should match snapshot 1`] = `
<div>
  <div
    className="nav"
  >
    <div
      className="container"
    >
      <div
        className="brand"
      >
        <a
          href="./"
        >
          iReporter
        </a>
      </div>
      <label
        htmlFor="toggle"
      >
        â˜°
      </label>
      <input
        id="toggle"
        type="checkbox"
      />
      <div
        className="menu"
      >
        <a
          href="./signup"
        >
          signup
        </a>
        <a
          href="#"
        >
          login
        </a>
      </div>
    </div>
  </div>
</div>
`;
